hl/test/test_image.c

```diff
@@ -665,10 +665,17 @@ test_generate(void)
     HL_TESTING2("make indexed image from sea data");
 
     for (i = 0; i < n_elements; i++) {
-        if (data[i] > 0)
+        if (data[i] > 0.0f)
             image_data[i] = 0;
-        else
-            image_data[i] = (unsigned char)((255 * (data[i] - xmin)) / xmin);
+        else {
+            assert(xmin < 0.0f); // never occurs
```

Comment:
We slap HD in front of all standard library calls. e.g., HDassert().

Reply:
Uh, I think you misunderstand, this is just test code illustrating the issue, not code to keep. :)